docs/pt/docs/deployment/concepts.md

```diff
@@ -0,0 +1,321 @@
+# Conceitos de Implantações
+
+Ao implantar um aplicativo **FastAPI**, ou na verdade, qualquer tipo de API da web, há vários conceitos com os quais você provavelmente se importa e, usando-os, você pode encontrar a maneira **mais apropriada** de **implantar seu aplicativo**.
+
+Alguns dos conceitos importantes são:
+
+* Segurança - HTTPS
+* Executando na inicialização
+* Reinicializações
+* Replicação (o número de processos em execução)
+* Memória
+* Etapas anteriores antes de iniciar
+
+Veremos como eles afetariam as **implantações**.
+
+No final, o objetivo final é ser capaz de **atender seus clientes de API** de uma forma **segura**, **evitar interrupções** e usar os **recursos de computação** (por exemplo, servidores remotos/máquinas virtuais) da forma mais eficiente possível. 🚀
```

Comment:
I would change:

"No final, o objetivo final..."

for something like:

"No final, o principal objetivo..."

It's odd to use final two times in a row